packages/salesforcedx-utils-vscode/test/jest/context/workspaceContextUtil.test.ts

```diff
@@ -56,6 +61,7 @@ describe('WorkspaceContext', () => {
   it('should load the default username and alias upon initialization', () => {
     expect(workspaceContextUtil.username).toEqual(testUser);
     expect(workspaceContextUtil.alias).toEqual(testAlias);
+    expect(reloadConfigAggregatorsMock).toHaveBeenCalled();
```

Comment:
if we mock something we should verify it is called when expected (or not called)